CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,83 @@
+# Code of Conduct
+
+## 1. Purpose
+
+A primary goal of AnitaB.org Open Source is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion (or lack thereof).
+
+This code of conduct outlines our expectations for all those who participate in our community, as well as the consequences for unacceptable behavior.
+
+We invite all those who participate in open source contributions to help us create safe and positive experiences for everyone.
+
+## 2. Open Source Citizenship
+
+A supplemental goal of this Code of Conduct is to increase open source citizenship by encouraging participants to recognize and strengthen the relationships between our actions and their effects on our community.
+
+Communities mirror the societies in which they exist and positive action is essential to counteract the many forms of inequality and abuses of power that exist in society.
+
+If you see someone who is making an extra effort to ensure our community is welcoming, friendly, and encourages all participants to contribute to the fullest extent, we want to know.
+
+## 3. Expected Behavior
+
+The following behaviors are expected and requested of all community members:
+
+*   Participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.
+*   Exercise consideration and respect in your speech and actions.
+*   Attempt collaboration before conflict.
+*   Refrain from demeaning, discriminatory, or harassing behavior and speech.
+*   Be mindful of your surroundings and of your fellow participants. Alert community leaders if you notice a dangerous situation, someone in distress, or violations of this Code of Conduct, even if they seem inconsequential.
+*   Remember that community event venues may be shared with members of the public; please be respectful to all patrons of these locations.
+
+## 4. Unacceptable Behavior
+
+The following behaviors are considered harassment and are unacceptable within our community:
+
+*   Violence, threats of violence or violent language directed against another person.
+*   Sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory jokes and language.
```

Comment:
suggestion: Can you also add ```casteist```?

Reply:
Interesting, instead of changing right here, can you bring this up on Zulip. These are versions that we are using on all projects. I just would like us to check with @mayburgos because if this is something we want to change here then we also want to change in other repos.

Reply:
Oh, 😅 I didn't know that. It was just a suggestion. I added a few more suggestions thinking that this CoC is specific to this repo. Otherwise, it is ready to merge 😄

Reply:
I actually think you're right in a lot of these changes. To be more inclusive to international issues and also updating the language so it's easier for non-native English speakers to understand clearly.